frontend/src/components/LoginPage.vue

```diff
@@ -47,14 +49,36 @@ export default defineComponent({
       const provider = new firebase.auth.GoogleAuthProvider();
       provider.addScope("profile");
       provider.addScope("email");
+      provider.addScope("https://www.googleapis.com/auth/calendar");
       firebase
         .auth()
         .signInWithPopup(provider)
         .then(function(result) {
-          if (result.credential) {
+          if (result.credential && result.user) {
             const credential = result.credential as firebase.auth.OAuthCredential;
             const token = credential.accessToken;
+            const firebaseID = result.user.uid;
+            const uname = result.user.displayName;
+            const email = result.user.email;
             console.log("OK - OAuth Token: " + token);
+            console.log(
+              `firebaseID: ${firebaseID}, uname: ${uname}, email: ${email}`
+            );
+            axios
+              .post("http://localhost:8000/createUserCalendar", {
```

Comment:
is this endpoint exposed anywhere?

Reply:
that's right, that's old, will remove it:)